### PR TITLE
FOR ALL INSTANCES OF "PERMANENT BAN" IN THE CODEBASE IT HAS BEEN SWITCHED TO "BAN UNTIL SUCCESSFUL APPEAL" BECAUSE IT IS MORE ACCURATE AND I SAW BAY DOING THIS AND FIGURED IT WOULD BE HANDY HERE TOO SINCE JUST ABOUT EVERY PERMA IS EITHER A 6 MONTH WAIT FOR YOU TO KISS ASS BAN OR JUST TO GET YOU TO EXPLAIN YOURSELF ON THE FORUMS THIS IS ALL IN THE TITLE BECAUSE APPARENTLY DESPITE GOING OVER THIS IN THE DESCRIPTION NO ONE CARES ENOUGH TO READ IT LOOKING AT YOU @PKPenguin12 AND @NikNakFlak 

### PR DESCRIPTION
**Pull request by Iamgoofball**
It's more accurate. Saw bay doing this, figured it'd be handy here too, since just about every perma is:
1. just 6 month bans until you ass-kiss hard enough
2. just to get you to explain yourself on the forums

### CODE REVIEW

**Comment by ChangelingRain** [comment]
Quite possibly a good idea.

**Comment by lzimann** [comment]
I am not a fan of this.

**Comment by DaxDupont** [comment]
"expires = " The is a ban until appeal.""
Is this a byond quirk or misspelled?

**Comment by Penguaro** [comment]
Another language possibility could be "Indefinite Ban" which suggests an uncertain end time (but the possibility for change). Just my two cents.

**Comment by ChangelingRain** [comment]
Maybe "ban until successful appeal."
Because it's not just any appeal, it's just a successful one.

**Comment by Iamgoofball** [comment]
Right

**Comment by PKPenguin321** [comment]
I'd like this added alongside permabans. This would be really useful for people who log out during investigations

**Comment by DaxDupont** [comment]
@PKPenguin321 Shouldn't it be enough to make this change and just use the ban reason for communicating that instead of making another option?

**Comment by lzimann** [comment]
We already use the ban reason for communicating this, or at least something along these lines.

**Comment by optimumtact** [comment]
Yeah but you're not consistent at it Lzi, this seems like a harmless change.

**Comment by Niknakflak** [comment]
I'll start using tempbans for 9999999999 minutes instead if this is merged

**Comment by optimumtact** [comment]
You can do whatever you like niknak

**Comment by FantasticFwoosh** [comment]
If it isn't clear enough you should probably add the URL to the appeals page too, or maybe thats too much hugboxing.

**Comment by Jordie0608** [comment]
I think for little gain this'll cause confusion with everything still referring to permabans i.e. DB, all admin tools, jobban messages, the code iteself, the very same lines that are being changed here.

**Comment by Iamgoofball** [comment]
This is for players front end only.

**Comment by Jordie0608** [comment]
Yes, so what admins and coders still refer to as "permabans" players will start calling "bans until appeal", except if it's a jobban.

**Comment by Iamgoofball** [comment]
Because in the end, that's what they're used for.

**Comment by bgobandit** [comment]
"Ban until appeal" is dumb because I guarantee there will be at least one rules lawyer saying "but the ban said it was only in place UNTIL I APPEALED IT"

**Comment by ChangelingRain** [comment]
@bgobandit I noted that thus my above comment about `Maybe "ban until successful appeal."`

**Comment by lzimann** [comment]
I think this will create more confusion than it's worth.

**Comment by Iamgoofball** [comment]
you're right

**Comment by Iamgoofball** [comment]
its done

File: code/modules/admin/IsBanned.dm [hunk 1]
```diff
@@ -103,7 +103,7 @@
 			if(text2num(duration) > 0)
 				expires = " The ban is for [duration] minutes and expires on [expiration] (server time)."
 			else
-				expires = " The is a permanent ban."
+				expires = " The is a ban until successful appeal."
```
Comment (MrStonedOne) [inline code/modules/admin/IsBanned.dm]: the word `The` should be `This`

You touched the line so you have to fix it.

**Comment by MrStonedOne** [comment]
you could instead say "This ban will not automatically expire and must be manually appealed. if you want to truly avoid confusion

**Comment by bgobandit** [comment]
or shorter, "This ban must be manually appealed."

**Comment by MrStonedOne** [comment]
no, that *implies* there is such a thing as an automatic appeal. and while that's what a ban expiring is from a purely technical viewpoint, it would be confusing to users with lesser ability to draw abstract connection then you and I.

My version is as wordy as it is because it draws the connection between automatic appeal and automatic expire for the user.

**Comment by Iamgoofball** [comment]
w o r d s

**Comment by F-OS** [comment]
Hi, it might be helpful if you add a way for admins to say when the ban can be appealed, for instance "This is a ban until successful appeal, You may appeal this in 3 weeks" Or "This is a ban until successful appeal, You may appeal this immediately" Or "This is a ban until successful appeal, This is a blacklist ban, it can only be removed by the current host"

**Comment by MrStonedOne** [comment]
In almost all cases, bans can be appealed immediately, the rare ones where this isn't the case can be handled in the reason, they don't happen enough to justify Yet Another Prompt™ on every ban.

**Comment by F-OS** [comment]
alright, good point, thanks mso.

**Comment by peoplearestrange** [comment]
Why though? This seems like needless busy work to be shown to be doing something "for the players".

Do we honestly expect people haven't appealed because "Its a permanent ban therefore why bother". And honestly if thats the case, why do we want them back? If someone actually WANTS to come back, they will try. Why would we want indifferent assholes?

I see this as the equivalent of "Contents maybe hot" on coffee cups. Sure its right, but why...

**Comment by Iamgoofball** [comment]
It's to cover their ass

**Comment by peoplearestrange** [comment]
Except we're not a huge corporation, we're a game. You've also ignored my other points and just gone for picking apart an analogy I added at the end.

**Comment by MrStonedOne** [comment]
>We take the wording from pretty much every gamer server ever. Do we honestly expect people haven't appealed because "Its a permanent ban therefore why bother".

Yes.

**Comment by KorPhaeron** [comment]
@peoplearestrange

Yes, especially for newer players who dont understand at all how our culture is.

**Comment by optimumtact** [comment]
We've literally seen appeals one or more years later from people who were permabanned just to explain on the forum.

**Comment by peoplearestrange** [comment]
Ok fair enough.
Again, do we add the "You can appeal this ban on the forums (LINK)" To perma bans? I know we do it for temp bans. If we don't then maybe that would be the first step?

**Comment by MrStonedOne** [comment]
That message is added if you are banned while playing, but the you can not connect you are banned screen doesn't have that link at all. Mainly because links on that screen don't work.

**Comment by Ausops** [comment]
If someone is getting perma'd, they should be at a point where they don't have any excuse for not knowing enough server culture and a point where they need to put in the effort to find out how to appeal and actually want to go through that effort despite being 'permanently' banned.

**Comment by optimumtact** [comment]
Except that you use permabans to make people come and explain on the forums, emu are you even switched on?

**Comment by Jordie0608** [comment]
Actually we stopped doing that now that we have player messages.

**Comment by peoplearestrange** [comment]
@MrStonedOne Thanks for explaining.

@optimumtact Jordie's correct, we should have moved away from bans for "explain on the forums". Plus to be honest those bans always did say "explain on the forums" as a ban reason, and that part you do get to see the reason.

The reason I feel like this is unnecessary is because I feel like we're over explaining something that "costs" us maybe 1 or 2 lazy players.
When PR's are made is there at least some forethought into thinking what is the actual return on this? Though to be honest we've all argued this so much no ones going to come up looking like time hasn't been wasted (including myself). But I still can't help but think this will make us seem petty.

**Comment by kilog** [comment]
what the fuck, man

**Comment by Iamgoofball** [comment]
@kilog I've been told that I need to be more descriptive with my PRs and apparently no body reads the description

**Comment by Iamgoofball** [comment]
take it up with our head administrator he asked for this

**Comment by PKPenguin321** [comment]
>h-he saw trough my meme title trick!!!
>F-FUCK Y-Y-YOU PKP A-AND NIKN-NAK
>autistic crying and screaming

**Comment by Iamgoofball** [comment]
honestly im more offended you really think I would intentionally fuck with the players that way

when did you become a more caustic cunt than me

**Comment by PKPenguin321** [comment]
Still lashing out because you've been called out I see

**Comment by peoplearestrange** [comment]
Have you always been like this or is it years of not getting things exactly your way thats made you this way?

Seriously, I enjoy your ideas and you've done a lot for the game, but you seem to really struggle to interact with other people, specially when they question you or ask for explanations.

**Pull request closed without merging**